Makefile

```diff
@@ -1,13 +1,21 @@
-JOBS ?= $(shell nproc)
-JOBS ?= 2
```

Comment:
This variable was only used by 'make go' below.  Passing -j when you invoke 'make' will apply it to all submakes (by switching to ${MAKE} below).  There isn't a particularly clean way to specify a default -j option. We could set MAKEFLAGS here but then changing the number of jobs at invocation time is trickier (need to override MAKEFLAGS env var instead of just passing -j).